### PR TITLE
add header, footer, static-pages

### PR DESCRIPTION
**Pull request by thanhgiong195**

### CODE REVIEW

**Comment by thanhgiong195** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
+      <%= csrf_meta_tags %>
+      <%= stylesheet_link_tag "application", media: "all",
+        "data-turbolinks-track": "reload" %>
+      <%= javascript_include_tag "application", "data-turbolinks-track": "reload" %>
   </head>
-
   <body>
-    <%= yield %>
+    <%= render "layouts/header" %>
+      <%= yield %>
```
Comment (dieunb) [inline app/views/layouts/application.html.erb]: indent

**Comment by dieunb** [comment]
commented

**Comment by thanhgiong195** [comment]
ready

**Pull request merged**